packages/www/hooks/use-api/index.tsx

```diff
@@ -61,12 +61,26 @@ const makeContext = (
       if (res.status === 204) {
         return [res];
       }
+
       // todo: not every endpoint will return JSON
       const body = await res.json();
       // todo: this can go away once we standardize on body.errors
       if (!Array.isArray(body.errors) && typeof body.error === "string") {
         body.errors = [body.error];
       }
+
+      const tokenExpired =
+        res.status === 401 && body.errors?.[0] === "access token expired";
+      if (tokenExpired && state.refreshToken && !isRetry) {
```

Comment:
Why do we need to check `isRetry` here?

Reply:
Because we retry the API call that failed on L80, and we don't want to have an infinite loop here trying to refresh the token in case something went wrong